op-node/rollup/derive/engine_controller.go

```diff
@@ -32,6 +32,8 @@ const (
 	syncStatusFinishedEL                // EL sync is done & we should be performing consolidation
 )
 
+const DefaultElTriggerGap = 3600
```

Comment:
What is the function of this constant value? The flag el-trigger.gap defalut value is 86400, but the DefaultElTriggerGap is 3600.

Reply:
removed since it not has been used